amnestywebsite/humanity-theme
# Capitalises the first letter of the language pop in banner for ES

Epic: https://github.com/amnestywebsite/humanity-theme/issues/417
Issue: https://github.com/amnestywebsite/amnesty-wp-theme/issues/2451

Capitalises the first letter of the language pop in banner for ES

**Steps to test**:
1. Open the /es/ site in an incognito browser
2. When the language pop in shows
3. Check the texts first letter is capitalised

**Considerations**:
- Localisation

Example: http://bigbite.im/i/zSOroO
